pkgs/applications/misc/stog/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, ocaml, findlib, ocf, ptime,
+  uutf, uri, ppx_blob, xtmpl, ocaml_lwt, higlo, camlp4, omd
+}:
+
+stdenv.mkDerivation rec {
+  name = "stog-${version}";
+  version = "0.17.0";
+  src = fetchFromGitHub {
+    owner = "zoggy";
+    repo = "stog";
+    rev = "release-${version}";
+    sha256 = "06fnl3im0rycn05w39adfmm7w4s8l3jrj43h8f8h3b56grh21x0d";
+  };
+
+  buildInputs = [ ocaml ocf ptime uutf uri ppx_blob xtmpl ocaml_lwt higlo camlp4 ];
+  propagatedBuildInputs = [ findlib omd ];
```

Comment:
How did you chose this list of propagated inputs? The `stog` library needs some more.